### PR TITLE
update release workflow using twine (docs, scripts), see #4213

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -333,9 +333,11 @@ Checklist:
   do a fresh start based on that.
 - run tox and/or binary builds on all supported platforms via vagrant,
   check for test failures
-- create a release on PyPi::
+- create sdist, sign it, upload release to (test) PyPi::
```
Comment (RonnyPfannschmidt) [inline docs/development.rst]: i do wonder if at a later point a follow-up for creating a "manylinux" based wheel might be sensible (statically linking the dependencies)

Comment (ThomasWaldmann) [reply docs/development.rst]: At a later point maybe, but not as long as that uses a ancient linux for building, which is below borg's minimum requirements.

Comment (RonnyPfannschmidt) [reply docs/development.rst]: i wonder if manylinux2010 is sufficient

Comment (RonnyPfannschmidt) [reply docs/development.rst]: hmm, i suspect centos6 may be too ancient

**Pull request merged**